src/style.css

```
@@ -1 +1,7 @@
-/* styles go here */
+.fieldspace:not(:last-child) {
+  margin-bottom: 10px;
+}
+
+fieldset {
```

use classes instead of tags
```suggestion
.fieldsetspace {
```